src/main/java/net/datafaker/providers/base/Twitter.java

```diff
@@ -15,7 +17,9 @@
  * @since 0.9.0
  */
 public class Twitter extends AbstractProvider<BaseProviders> {
-
+    private static final Logger LOGGER = Logger.getLogger(Twitter.class.getName());
```

Comment:
Full disclosure I only have experience with Log4j. Is a string needed here or is passing class fine?

Reply:
In case of `java.util.logging.Logger` it requires `String`. To be sure it is unique usually people provide classnames